test/cases/compile_errors/runtime_operation_in_comptime_block.zig

```diff
@@ -0,0 +1,17 @@
+const std = @import("std");
+
+test "comptime expression" {
+    comptime {
+        std.log.warn("hello in comptime", .{});
+    }
+}
+
+// error
+// backend=stage2
+// target=native
+//
+// :110:38: error: unable to evaluate comptime expression
+// :110:60: note: operation is runtime due to this operand
+// :139:21: note: called from here
+// :190:16: note: called from here
+// :5:21: note: called from here
```

Comment:
Do not use actual numbers when referencing lines in the standard library.
```suggestion
// :?:?: error: unable to evaluate comptime expression
// :?:?: note: operation is runtime due to this operand
// :?:?: note: called from here
// :5:21: note: called from here
```

Reply:
Ah, interesting. Didn't know you could do that.

Reply:
Doesn't quite seem to be working in this case:
```
compiler [599/1183] compile_errors/runtime_operation_in_comptime_block (stage2, native) [1/1] u... 
Expected error not found:
================================================================================
:?:?: error: unable to evaluate comptime expression
================================================================================
Expected error not found:
================================================================================
:?:?: note: operation is runtime due to this operand
================================================================================
Expected error not found:
================================================================================
:?:?: note: called from here
================================================================================
Expected error not found:
================================================================================
:5:21: note: called from here
================================================================================
update_index=0
```
Even though I can definitely find ":?:?: error: unable to evaluate comptime expression" in there... do you have any guesses?

Reply:
It's because tests are not analyzed when using `build-obj` either add `//is_test=1` or remove the test block around the comptime block (preferable).

Reply:
Ah, I see! Yeah the `test` does not seem necessary.

Reply:
And yet it still fails like this:
```
compiler [599/1183] compile_errors/runtime_operation_in_comptime_block (stage2, native) [1/1] u... 
Expected error not found:
================================================================================
:4:17: note: called from here
================================================================================
update_index=0
```
Maybe I still need `//is_test=1`?

Reply:
There are a lot more notes when not testing and the error comes from a different place so maybe.

Reply:
That's a good point. Alright, I'll try.